Microsoft.Toolkit.Uwp.UI.Controls/Loading/Loading.xaml

```diff
@@ -0,0 +1,112 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    
+    <Style TargetType="controls:Loading">
+        <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
```

Comment:
Change these values to center.
Add HorizontalAlignment and VerticalAlignment properties and set them to Stretch